app/containers/ViewProfilePage/ReferralProgram.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import H3 from 'components/H3';
+import commonMessages from 'common-messages';
+import * as routes from 'routes-config';
+import { FormattedMessage } from 'react-intl';
+
+import profileMessages from '../Profile/messages';
+import { BaseStyled } from './SettingsOfUser';
+
+const ReferralProgram = ({ className, user, writeToBuffer }) => {
+  const referralLink = `https://peeranha.io/users/${user}`;
```

Comment:
Is referral link is the same as user profile page?

Referal link should be a link to a home page with query string parameter:
https://peeranha.io?referralCode=<refcode>

Reply:
Okay, I just didn't know how the link should look